src/view.js

```diff
@@ -58,86 +76,125 @@ export class View {
+    const gameDescription = this.#createElement('h3');
+
+    gameDescription.append(
+      'Guess the interval ',
+      this.#createElement('wbr'),
+      'between the two tones.'
     );
     const gameStartButton = this.#createButton('Start');
+    gameStartButton.classList.add(classNames.primaryButton);
 
-    this.appContainer.append(gameTitle, gameRuleParagraph, gameStartButton);
+    this.appContainer.append(gameTitle, gameDescription, gameStartButton);
     gameStartButton.addEventListener('click', this.#publishGameStartEvent);
   }
 
   renderQuestionPage() {
+    this.appContainer.classList.remove(classNames.centerVertically);
+    this.appContainer.classList.add(classNames.flowSpaceLarge);
     this.appContainer.replaceChildren();
 
+    // Header: current question number, score
     const currentQuestionNumberDisplay = this.#createElement(
-      'div',
+      'h3',
       'Question: '
     );
     const currentQuestionNumberSpan = this.#createElement('span');
     currentQuestionNumberSpan.id = 'questionNumber';
     currentQuestionNumberDisplay.append(currentQuestionNumberSpan);
 
+    const currentScoreDisplay = this.#createElement('h3', 'Score: ');
+    const currentScoreSpan = this.#createElement('span');
+    currentScoreSpan.id = 'currentScore';
+    currentScoreDisplay.append(currentScoreSpan);
+
+    const headerContainer = this.#createElement('div');
+    headerContainer.classList.add(classNames.header);
+    headerContainer.append(currentQuestionNumberDisplay, currentScoreDisplay);
+
+    // Question: button to (re)play tones, game instruction
     const playTonesButton = this.#createButton('Play tones');
     playTonesButton.id = 'playTonesBtn';
+    playTonesButton.classList.add(classNames.primaryButton);
     const gameRuleParagraph = this.#createElement(
       'p',
       'Guess the interval between the 2 tones.'
     );
 
-    const buttonsGridContainer = this.#createElement('div');
-
-    const submitAndMoveToNextQuestionButton =
-      this.#createButton('Move to next');
-    submitAndMoveToNextQuestionButton.disabled = true;
-    const skipQuestionButton = this.#createButton('Skip');
-
-    const currentScoreDisplay = this.#createElement('div', 'Score: ');
-    const currentScoreSpan = this.#createElement('span');
-    currentScoreSpan.id = 'currentScore';
-    currentScoreDisplay.append(currentScoreSpan);
-
-    this.appContainer.append(
-      currentQuestionNumberDisplay,
-      playTonesButton,
-      gameRuleParagraph,
-      buttonsGridContainer,
-      skipQuestionButton,
-      submitAndMoveToNextQuestionButton,
-      currentScoreDisplay
-    );
-
+    // Answer: interval buttons, guess button, skip button
     const intervalButtons = Object.entries(intervals).map(
       ([intervalName, semitones]) => {
         const button = this.#createButton(intervalName);
+        button.classList.add(classNames.secondaryButton);
+        button.dataset.semitones = semitones;
```

Comment:
I kind of have seen how to use data attribute in CSS, but never knew how to set it in JS by using dataset. Thank you for using this so that I can learn about this! ❤️